endless-mount-dirs.service

```diff
@@ -25,9 +25,10 @@ RequiresMountsFor=/var
 # Only run if the directories don't exist
 ConditionPathIsDirectory=|!/var/endless
 ConditionPathIsDirectory=|!/var/endless-extra
+ConditionPathIsDirectory=|!/var/workdir
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=/bin/mkdir -p /var/endless /var/endless-extra
+ExecStart=/bin/mkdir -p /var/endless /var/endless-extra /var/workdir
 ExecStart=-/bin/chown app-manager:app-manager /var/endless /var/endless-extra
```

Comment:
Thinking again, I don't think the workdir ownership should be changed. It's entirely for the kernel's internal use, so I don't think we should allow the app-manager user to accidentally write files into it. So, just changing the workdir name should be enough for this.